.circleci/config.yml

```diff
@@ -289,6 +289,7 @@ jobs:
           command: |
             $env:OKTETO_PATH="$($HOME)\project\artifacts\bin\okteto.exe"
             $env:Path+=";$($HOME)\project\artifacts\bin"
+            $env:SSH_AUTH_SOCK = (Get-Command ssh-agent).Definition -replace 'ssh-agent.exe','ssh-agent.sock'
```

Comment:
Why is needed this now? For the new tests we added?

Reply:
@ifbyol this is needed to make remote deploy work from CI. It's already set at line [281](https://github.com/okteto/okteto/pull/4177/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R281) for the `deploy` tests. Now I am adding a test for `okteto up` + remote deploy so we also need it here.

Without this, Windows Integration tests fail in CI with an error related to `SSH_AUTH_SOCK`

Reply:
Nice! Thanks for the context!